rules/prefer-exponentiation-operator.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+const getDocsUrl = require('./utils/get-docs-url');
+
+const isMathPow = node => {
+	const {callee} = node;
+	return (
+		callee.type === 'MemberExpression' &&
+		callee.object.type === 'Identifier' &&
+		callee.object.name === 'Math' &&
+		callee.property.type === 'Identifier' &&
+		callee.property.name === 'pow'
+	);
+};
+
+const parseArgument = (context, arg) => {
+	if (arg.type === 'Identifier') {
+		return arg.name;
+	}
+
+	return context.getSourceCode().getText(arg);
+};
+
+const create = context => {
+	return {
+		CallExpression(node) {
+			if (isMathPow(node)) {
+				context.report({
+					node,
+					message: 'Prefer the exponentiation operator over `Math.pow()`.',
+					fix: fixer => {
```

Comment:
I would move the fixer method to a top-level function too, for readability.

Reply:
Since you need `fixer`, `context` and `node` to run the fix, I'm not sure moving it out really helps much. I gave it a shot; let me know what you think.